packages/uni_lint/lib/uni_lint.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:analyzer/error/listener.dart';
+import 'package:custom_lint_builder/custom_lint_builder.dart';
+import 'package:analyzer/dart/ast/ast.dart';
+
+PluginBase createPlugin() => _ExampleLinter();
+
+class _ExampleLinter extends PluginBase {
+  @override
+  List<LintRule> getLintRules(CustomLintConfigs configs) => [
+        StringLiteralsLint(),
+      ];
+}
+
+class StringLiteralsLint extends DartLintRule {
```

Comment:
Use a more descriptive name, such as `NoStringLiteralsInWidgetsLint`
Also, change the name of the rule accordingly